src/nvme/private.h

```diff
@@ -45,6 +45,7 @@ struct nvme_ns {
 	int fd;
 	__u32 nsid;
 	char *name;
+	char *chrdev_name;
```

Comment:
I am not sure if chdrev_name (and also nvme_ns_get_chrdev_name) is a good name. The existing nvme device is also a char device. The differenece is this one is the 'generic' one. So I think the generic bit should be somehow included in the naming scheme.

Reply:
let me have this `generic_name`.